api.yaml

```diff
@@ -161,6 +161,27 @@ paths:
           description: Invalid payload.
         '500':
           description: Profile cannot be updated.
+  "/api/v1/notify":
+    x-swagger-router-controller: NotificationController
+    parameters:
+      - in: body
+        name: body
+        schema:
+          $ref: "#/definitions/Notification"
+        x-examples:
```

Comment:
perhaps the example should be in the definition of `Notification` ?

Reply:
For others endpoint the example is in the endpoint definition and not in the refs definition. Maybe we could open a story to change this behaviour?

Reply:
ok let's keep it consistent with the rest for now